CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ and let us know if it's not up-to-date (even better, submit a PR with your  corr
   - [Pull Request Checklist](#pull-request-checklist)
   - [Step 1: Open Issue](#step-1-open-issue)
   - [Step 2: Design (optional)](#step-2-design-optional)
-  - [Step 3: Work your Magic](#step-3-work-your-magic)
+  - [Step 3: Work your Magic](#step-3-work-your-magic)S
```

Comment:
This change is probably accidental, right?

Reply:
yeah, sorry.
cleaned up.